tests/test_cases/input/eof_after_escape.kdl

```diff
@@ -1 +1 @@
-node \
+node \
```

Comment:
Missed this case, NICE

Reply:
oh? You mean this failed in kdl-py? haha nice

Reply:
Yeah this also failed in ckdl :-)